src/components/Form/components/CSForm.tsx

```diff
@@ -78,7 +87,11 @@ const CSForm: FC<ICSForm> = ({ onSubmit, onError, children, ...props }) => {
         event: FormEvent<HTMLFormElement>
     ): void => {
         if (!onSubmit) return;
-        handleSubmit(onSubmit, onError)(event);
+        if (!actionData) {
+            handleSubmit(onSubmit, onError)(event);
+        } else {
+            onSubmit(undefined, event);
+        }
```

Comment:
Thoughts on making it such that if there are no errors it submits back to the server for server sided validation? At least that's how I intend to use it.